test/fakeintake/server/server.go

```diff
@@ -247,16 +256,34 @@ func (fi *Server) handleGetPayloads(w http.ResponseWriter, req *http.Request) {
 		})
 		return
 	}
+	format := req.URL.Query().Get("format") // raw or json
+	if format == "" {
+		format = "raw"
+	}
```

Comment:
As `format` is used only at line 269, what do you think about having `if req.URL.Query().Get("format") == "raw"` at line 269?